data-service/src/main/java/com/amazonaws/blox/dataservice/repository/model/EnvironmentDDBRecord.java

```diff
@@ -38,20 +38,21 @@
 @NoArgsConstructor
 public class EnvironmentDDBRecord {
 
-  public static final String ENVIRONMENT_ID_HASH_KEY = "environmentId";
+  public static final String ACCOUNT_ID_CLUSTER_HASH_KEY = "accountIdCluster";
```

Comment:
Why do we change the hash key from environmentId into accountIdCluster? Is it likely that under the same accountId and cluster there are multiple environments with different environment names?

Reply:
So this was always composed of accountId and cluster but it was confusing because it was called environmentId and did not match the environmentId of the environment object.

Reply:
Yes that's true. So what we really want as the key here is not the actual EnvironmentId object. Thank you for clarifying this :-D